carbonmark-api/src/utils/helpers/listings.utils.ts

```diff
@@ -31,12 +22,27 @@ export const getCreditListings = async (
   });
   return project;
 };
+const isActiveListing = (
+  l: {
+    active?: boolean | null;
+    deleted?: boolean | null;
+    leftToSell?: string | null;
+  },
+  minSupply?: number
+) => !!l.active && !l.deleted && Number(l.leftToSell) >= (minSupply || 0);
+
+/**
+ * For marketplace subgraph projects
+ * Returns true if project has an active, unexpired listing
```

Comment:
This comment is incorrect as `isActiveListing` is only checking for deletion, in other words a listing can be `active: true` and `deleted: false` but also be expired (expiration timestamp < current timestamp)

Reply:
I updated the comment in b46e71311
Expired listings are already filtered at gql query level